apis/logging/v1beta1/collector_config_types.go

```diff
@@ -59,6 +63,20 @@ type CollectorConfigList struct {
 	Items           []CollectorConfig `json:"items"`
 }
 
+func CollectorConfigCRD() (*crd.CRD, error) {
```

Comment:
Can we add the monitoring agent types as well to the wrangler pattern?

Reply:
We can, they weren't in main when I opened this though.  If they've been merged I will rebase

Reply:
yeah they are in main as of tuesday I believe